Benachrichtigung/module.php

```diff
@@ -333,22 +439,36 @@ public function SetNotifyLevel(int $Level)
 
                     $message = $action['message'];
                     if ($action['messageVariable'] !== 0) {
-                        $message .= strval(GetValue($action['messageVariable']));
+                        $message = str_replace('{variable}', strval(GetValue($action['messageVariable'])), $message);
                     }
+
                     switch ($action['actionType']) {
                         case self::SCRIPT_ACTION:
                             IPS_RunScriptEx($action['recipientObjectID'], ['RECIPIENT' => $action['recipientAddress'], 'TITLE' => $action['title'], 'MESSAGE' => $action['message'], 'MESSAGE_VARIABLE' => $action['messageVariable']]);
                             break;
 
                         case self::PUSH_NOTIFICATION_ACTION:
-                            WFC_PushNotification($action['recipientObjectID'], $action['title'], $message, 'alarm', $this->GetIDForIdent('ResetScript'));
+                            $targetID = $this->GetIDForIdent('ResetScript');
```

Comment:
Man könnte das ResetScript ja auch abschaffen und wenn AdvancedResponse aus ist, in die ResponseAction Variante nur ein generisches "Quittieren" Element einfügen. Dann kann das Skript weg und die TargetID lässt man immer auf InstanceID